.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,56 @@
+name: Integration Tests
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  selenium-tests:
+    name: Build the project
+    strategy:
+      fail-fast: false
+      matrix:
+        wp_service: [ 'wp59-php74', 'wp61-php74', 'wp61-php82' ]
+    runs-on: ubuntu-latest
+    env:
+      PROJECT: wordpress-plugin
+      ORGANIC_DEMO_SITE_APIKEY: ${{ secrets.ORGANIC_DEMO_SITE_API_KEY }}
+      COMPOSER_AUTH: '{"github-oauth":{"github.com":"${{ secrets.GITHUB_TOKEN }}"}}'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        id: python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Configure poetry
+        run: |
+          poetry self add poetry-dotenv-plugin
+          poetry install
+
+      - name: Cache Composer dependencies
```

Comment:
I suppose I should cache poetry deps too, which will save us maybe 10-15 seconds. I'll do this tomorrow (Thursday).